docs/development/journalist_api.rst

```diff
@@ -0,0 +1,414 @@
+Journalist Interface API
+========================
+
+This document describes the endpoints for SecureDrop's Journalist Interface
+API.
+
+Versioning
+~~~~~~~~~~
+
+The API is versioned and we are currently using version 1. This is set via the
+base URL, which is:
+
+.. code:: sh
+
+  /api/v1/
+
+Content Type
+~~~~~~~~~~~~
+
+Clients should send the following headers:
+
+.. code:: sh
+
+  'Accept': 'application/json',
+  'Content-Type': 'application/json'
+
+Authentication
+~~~~~~~~~~~~~~
+
+``POST /api/v1/token`` to get a token with the username, password, and 2FA
+token in the request body:
+
+.. code:: sh
+
+  {
+  	"username": "journalist",
+  	"password": "mypasswordgoeshere",
```

Comment:
Related to #2176, I think we should use "passphrase" everywhere because it emphasizes that the "secret thing" is not just a word but something complex and long.

Reply:
true, I should also use diceware in the docs for consistency... fixin'

Reply:
Oh dear. I think I need to be clearer in my comments.

I meant that the field should be named `passphrase` because I think (yes, def *an opinion*) that it is generally better to use passphrase over password because it emphasizes complexity.